samples/batch/js/node/main.js

```diff
@@ -0,0 +1,333 @@
+          console.log(jsonParsed.status);
+          var trans_status = jsonParsed.status;
+
+          if (trans_status == "Failed" || trans_status == "Succeeded")
+          {
+            completed = "True"
+          }
+        if (trans_status == "Succeeded")
+          {
+           
+          }
+        else if (trans_status == "Failed")
+            console.log("Transcription failed: {transcription.properties.error.message}")
+
+        }
+      };
+
+ var callback = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+   console.log('API called successfully. Returned data: ' + JSON.stringify(data));
+
+  }
+};
+
+var callbackGetTrans = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+   console.log('Callback Get API called successfully. Returned data: ' + JSON.stringify(data));
+    completed=false ;
+    var tid = data.self.substring(data.self.lastIndexOf("/")+1);
+    console.log("Trans Id " + tid);
+    status = _checkComplete(tid);
+
+    if (status == "Succeeded")
+    {
+        var opts = _getTranscriptionFiles();
+        apiInstance.getTranscriptionFiles(tid,opts,callbackGetTransFiles);
+    }
+    else if(data.status == "Failed")
+           console.log("Transcription failed: {transcription.properties.error.message}")
+  }
+};
+
+var callbackGetTransFiles = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+   console.log('callbackGetTransFiles API called successfully. Returned data: ' + JSON.stringify(data));
+   pag_files = data.values;
+   console.log(pag_files.length);
+   for (var i = 0; i < pag_files.length; i++){
+     var file_data = pag_files[i];
+     var url = file_data.links.contentUrl;
+    
+    console.log (" url " + url);
+     if(file_data.kind!='Transcription')
+        continue;
+     else{
+        request(url, { json: true }, (err, res, body) => {
+            if (err) { 
+                return console.log(err); 
+            }
+         //   console.log("***************************************************************  response "+JSON.stringify(body));
+            console.log("***************************************************************************");
+
+            console.log(body);
+            //content.decode('utf-8')
+        });
+        
+     }
+   }
+  }
+};
+        
+//_transcribe();
+//_deleteAllTranscriptions();
+
+function _transcribe()
+{
+    
+    // Configure API key authorization: apiKeyHeader
+    var apiKeyHeader = defaultClient.authentications['apiKeyHeader'];
+    apiKeyHeader.apiKey = '';
+    // Configure API key authorization: apiKeyQuery
+    var apiKeyQuery = defaultClient.authentications['apiKeyQuery'];
+    apiKeyQuery.apiKey = '';
+
+    //Create API Instance
+    //var apiInstance = new SpeechToTextApiV30.DefaultApi();
+
+    const subscriptionKey = "{your-subscription-key-here}";
+    const serviceRegion = "centralindia"; // e.g., "westus"
```

Comment:
use the constants defined at the top. Also elsewhere.

Reply:
@chlandsi - Have incorporated the changes in the latest code for main.js.